ballerina/context.bal

```diff
@@ -97,6 +99,35 @@ public isolated class Context {
     isolated function getFileInfo() returns map<Upload|Upload[]> = @java:Method {
         'class: "io.ballerina.stdlib.graphql.runtime.engine.EngineUtils"
     } external;
+
+    public isolated function resolve() returns anydata|error {
```

Comment:
The API should be `resolve(Field 'field) returns anydata`. It will return null if the field resolution returns an error.